docs/api/main-config/main-config.mdx

```diff
@@ -7,7 +7,10 @@ sidebar:
 
 The main configuration defines a Storybook project's behavior, including the location of stories, addons to use, feature flags, and other project-specific settings.
 
-## `main.js` or `main.ts`
+
+## The main configuration file: `main.js` or `main.ts`
+
+> **NOTE:** This file must be valid ESM, meaning it must use `import` instead of `require`, and neither `__dirname` nor `__filename` are available.
```

Comment:
```suggestion
<Callout variant="warning">

This file must be valid ESM. In other words, it must use `import` instead of `require`, and neither `__dirname` nor `__filename` are available.

</Callout>
```